app/assets/javascripts/tylium/modules/server_auto_save.js

```diff
@@ -0,0 +1,74 @@
+// Server Auto Save module is initialized on elements with
+// data-behavior=server-auto-save. This module creates an action cable
+// subscription bound to the editor it's initialized from. The socket will
+// accept content for resources to update, and create revisions from. It will
+// receive timestamps back from the server to update the `original_updated_at`
+// field which helps us with conflict resolution.
+(function($, window) {
+  function ServerAutoSave(form) {
+    this.form = form;
+
+    this._doneTypingInterval = 500;
+    this.originalUpdatedAt = form.querySelector('[data-behavior~=updated-as-auto-save]');
+    this.projectId = form.dataset.asProjectId;
+    this.resourceId = form.dataset.asResourceId;
+    this.resourceType = form.dataset.asResourceType;
+
+    this.init();
+  }
+
+  ServerAutoSave.prototype = {
+    init: function() {
+      var that = this;
+      this.editorChannel = window.App.cable.subscriptions.create({
+        channel: 'EditorChannel',
+        project_id: this.projectId,
+        resource_id: this.resourceId,
+        resource_type: this.resourceType
+      },{
+        connected: function() {
+          console.log('Subscribed to EditorChannel');
```

Comment:
Do we need some `<meta>` tag that lists the `Rails.environment` so we can disable/guard all these `console.log` calls to the Dev environment?

Reply:
I'm okay to drop the console logs. They were in the last WS we used, so I added them here to stay consistent.

Reply:
I'm commented out the console log messages but left the empty functions for future reference.